docs/sophia_features.md

```diff
@@ -134,6 +134,63 @@ main contract IntHolderFactory =
 In case of a presence of child contracts (`IntHolder` in this case), the main
 contract must be pointed out with the `main` keyword as shown in the example.
 
+### Contract interfaces and polymorphism
+
+Contracts can implement one or multiple interfaces, the contract has to define
+every entrypoint from the implemented interface and the entrypoints in both
+the contract and implemented interface should have the exact same type.
```

Comment:
```suggestion
Contracts can implement one or more interfaces. For that, a contract has to implement
every entrypoint declared in that interface. The implementation has to match the type of the declaration in the interface.
```

...does it really need to be exact? Can't it be a supertype?

Reply:
It can be  a supertype, should I mention that?

Reply:
Of course

Reply:
Changed here 41c0609837588db08bb611cb2c19380216381e68